packages/trader/src/Modules/CFD/Containers/trading-password-manager.jsx

```diff
@@ -49,9 +49,18 @@ ChangePassword.propTypes = {
     platform: PropTypes.string,
 };
 
-const PasswordReset = ({ email, platform }) => {
+const PasswordReset = ({ email, platform, account_group }) => {
     const onClickSendEmail = React.useCallback(() => {
-        const redirect_to = platform === CFD_PLATFORMS.MT5 ? 1 : 2;
+        let redirect_to = platform === CFD_PLATFORMS.MT5 ? 1 : 2;
+
+        // if account type is real convert redirect_to from 1 or 2 to 10 or 20
+        // and if account type is demo convert redirect_to from 1 or 2 to 11 or 21
+        if (account_group === 'real') {
+            redirect_to = Number(`${redirect_to}0`);
+        } else if (account_group === 'demo') {
+            redirect_to = Number(`${redirect_to}1`);
+        }
```

Comment:
Same duplicated code

Reply:
@matin-binary because we have 2 different places for changing passwords (in the dashboard and in settings page) so we need this logic in both places